bfcfg

```diff
@@ -465,9 +465,12 @@ boot_cfg()
       NIC_P0|NIC_P1)
         [ ! -d /dev/mst ] && mst start
         mac=$(flint -d /dev/mst/mt41682_pciconf0 q | grep "^Base MAC" | awk '{print $3}')
-        if [ -z "${mac}" ]; then
-          log_msg "boot: failed to get MAC for ${entry}"
-          continue
+        if [ -z "${mac}" ] || [ ."${mac}" = ."N/A" ]; then
+          mac=$(flint -d /dev/mst/mt41686_pciconf0 q | grep "^Base MAC" | awk '{print $3}')
```

Comment:
Question: Why are we using a different device here?

I understand that BlueField and Viper show different MST devices, should we check whether the device is present first? before calling flint. Later on, I think we may check the MAC address. Am I missing something?

Reply:
Done. Use '2>/dev/null' which seems more robust in case flint is not there.